version.json

```diff
@@ -1,7 +1,6 @@
 {
   "version": "15.1-preview",
   "publicReleaseRefSpec": [
-    "^refs/heads/master$",
-    "^refs/heads/xplat$"
+    "^refs/heads/.*-rtw"
```

Comment:
We could chat about branching patterns that work well for other repos I've seen, if you're interested.